File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int[] coins = {25, 10, 5, 1};

        int t = Integer.parseInt(br.readLine());

        for (int tc = 0; tc < t; tc++) {
            int c = Integer.parseInt(br.readLine());

            StringBuilder answer = new StringBuilder();


            for (int coin : coins) {
                if (c >= coin) {
                    answer.append(c / coin + " ");
                    c %= coin;
                } else {
                    answer.append(0 + " ");
                }
            }

            System.out.println(answer);
        }
    }
}

File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.py
t = int(input())
coins = [25, 10, 5, 1]
m_list = []  # 거스름 돈

for _ in range(t):
  m_list.append(int(input()))

for m in (m_list):
  result = []
  for coin in (coins):
    result.append(m // coin)
    m %= coin

  print(f'{result[0]} {result[1]} {result[2]} {result[3]}')